### ISSUE TITLE
missing libiomp5.dylib in PyTorch 1.10.2-1.5.7 

### ISSUE THREAD

**Issue by doudouyzgf** [bytedeco/javacpp-presets#1225]
Hey there,

Not sure if anyone else has run into this issue

```
Caused by: java.lang.UnsatisfiedLinkError: /.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/libjnitorch.dylib: dlopen(/.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/libjnitorch.dylib, 0x0001): Library not loaded: @rpath/libiomp5.dylib
  Referenced from: /.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/libtorch.dylib
  Reason: tried: '/Users/runner/work/javacpp-presets/javacpp-presets/pytorch/cppbuild/macosx-x86_64/lib/libiomp5.dylib' (no such file), '/.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/libiomp5.dylib' (no such file), '/.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/./libiomp5.dylib' (no such file), '/Users/runner/work/javacpp-presets/javacpp-presets/pytorch/cppbuild/macosx-x86_64/pytorch/torch/lib/libiomp5.dylib' (no such file), '/Users/runner/.javacpp/cache/openblas-0.3.19-
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1950)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1832)
	at java.lang.Runtime.load0(Runtime.java:811)
	at java.lang.System.load(System.java:1088)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:1747)
```

When I compared the jars for `1.12.1-1.5.8-SNAPSHOT` vs `1.10.2-1.5.7` on my local machine, I did find `libiomp5.dylib` is available in the former, but not in the latter. 

```bash
$ ls -lth ~/.javacpp/cache/pytorch-1.12.1-1.5.8-SNAPSHOT-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/lib
libc10.dylib         libgfortran.dylib    libjnitorch.dylib    libquadmath.0.dylib  libtorch_cpu.dylib
libgfortran.5.dylib  libiomp5.dylib       libopenblas.0.dylib  libtorch.dylib


$ ls -lth ~/.javacpp/cache/pytorch-1.10.2-1.5.7-macosx-x86_64.jar/org/bytedeco/pytorch/macosx-x86_64/lib
libc10.dylib         libgfortran.4.dylib  libjnitorch.dylib    libquadmath.0.dylib  libtorch_cpu.dylib
libgcc_s.1.dylib     libgfortran.dylib    libopenblas.0.dylib  libtorch.dylib
```

Another related question: is there a plan to release 1.5.8 and if yes when? 

Many thanks in advance!

**Comment by saudet** [comment]
Fixed in 19f61bdb01df810eb0c4d0dbbcebfdb9654e59c8
> Another related question: is there a plan to release 1.5.8 and if yes when?

Probably sometime this fall

**Comment by doudouyzgf** [comment]
Hi @saudet , any chance of 1.5.8 being released soon? :)

**Comment by saudet** [comment]
The fix has been released with version 1.5.8. Enjoy!
